### PR TITLE
[Complete][Phase1]

### PR DESCRIPTION
**Pull request by TechAlchemy423**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208239081014545

### CODE REVIEW

**Comment by TechAlchemy423** [comment]
**- should use these environmental variables in your .env file.**

  DEEPGRAM_STT_DOMAIN=https://api.deepgram.com/
  DEEPGRAM_SOCKET_URL=wss://sts.sandbox.deepgram.com/agent
  DEEPGRAM_API_KEY=b6f78efae4b2128e4c3d8f81fb6c740b8c0bc077
  DEEPGRAM_ENV=development
  JWT_SECRET=my-secret
  NEXT_PUBLIC_DEEPGRAM_SOCKET_URL=wss://sts.sandbox.deepgram.com/agent
  NEXT_PUBLIC_DEEPGRAM_API_KEY=b6f78efae4b2128e4c3d8f81fb6c740b8c0bc077
  
 **- The UI is changed from original UI cuz of using Agent API. So plz check the UI and if you want to maintain original UI, it will take  2 days.**
 
 **- I counted time for this work as 40 hrs.**
 
 This is Deepgram project and I need to read Deepgrap  documentation to get clear understand about this API logic and integration police 

Previously I do not know about this Deepgrap because it is not a popular framwork like React.js or Next.js. This Deepgram is needed only for this project So I  took some time like 2 days extra for learning. Because I work and read this documentation that I needed to work . 

It is take only 4/5 days But I take some extra for this reason.

File: package.json
Comment (lukeocodes) [inline package.json:1]: @ricky0123/vad-react, onnxruntime-web and openai should all have been removed by now

File: app/lib/constants.ts [hunk 1]
```diff
@@ -132,3 +132,117 @@ export const greetings = [
 ];
 
 export const silentMp3: string = `data:audio/mp3;base64,SUQzBAAAAAABEVRYWFgAAAAtAAADY29tbWVudABCaWdTb3VuZEJhbmsuY29tIC8gTGFTb25vdGhlcXVlLm9yZwBURU5DAAAAHQAAA1N3aXRjaCBQbHVzIMKpIE5DSCBTb2Z0d2FyZQBUSVQyAAAABgAAAzIyMzUAVFNTRQAAAA8AAANMYXZmNTcuODMuMTAwAAAAAAAAAAAAAAD/80DEAAAAA0gAAAAATEFNRTMuMTAwVVVVVVVVVVVVVUxBTUUzLjEwMFVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVf/zQsRbAAADSAAAAABVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVf/zQMSkAAADSAAAAABVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV`;
+
+/**
+ * Voice Options
+ */
+export const voices: {
```
Comment (lukeocodes) [inline app/lib/constants.ts:139]: Remove these options, keep the app simple for now

File: app/lib/constants.ts [hunk 2]
```diff
@@ -132,3 +132,117 @@ export const greetings = [
...
+    name: "Zeus",
+    avatar: "/aura-zeus-en.svg",
+    language: "English",
+    accent: "US",
+  },
+};
+
+// models
+
+export const models = [
```
Comment (lukeocodes) [inline app/lib/constants.ts:223]: Remove these too, we keep it very simple

File: app/components/Conversation.tsx
Comment (lukeocodes) [inline app/components/Conversation.tsx:1]: This file isn't even used anymore

**Review by lukeocodes — CHANGES_REQUESTED** [review]

**Review by lukeocodes — APPROVED** [review]

**Pull request merged**